frontend/src/global.js

```diff
@@ -1,3 +1,5 @@
-const BackendURL = 'http://127.0.0.1:5000/mortgageCalculator';
-const wiborChartURL = 'http://127.0.0.1:5000/wiborChart';
+const BackendURL =
+  'https://monkfish-app-i4ovz.ondigitalocean.app/docker-backend/mortgageCalculator';
+const wiborChartURL =
+  'https://monkfish-app-i4ovz.ondigitalocean.app/docker-backend/wiborChart';
```

Comment:
I'm not quite sure whether this is a correct way to hardcode URL for frontend endpoints for DigitalOcean
I guess looking at a documentation my suggestion here would be to introduce it as a docker image environment variable: https://docs.digitalocean.com/products/app-platform/how-to/use-environment-variables/
But as long as it works it's fine with me, and you can ignore this comment